### PR TITLE
Add admin api update user's standard attributes doc

### PR DESCRIPTION
**Pull request by carmenlau**

### CODE REVIEW

File: apis/admin-apis/update-users-standard-attributes.md [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+## Overview
+
+To update the user's standard attributes:
+
+1. Fetch the user's original standard attributes by query first.
+2. Change the attributes payload and update the user's standard attributes by mutation.
+
+## API Details
+
+### Generate the node ID for fetching the user object
```
Comment (louischan-oursky) [inline apis/admin-apis/update-users-standard-attributes.md:12]: I think this section is worth its own entry under the sidebar. Maybe put it as the first item under "Admin APIs"?

Comment (fungc-io) [reply apis/admin-apis/update-users-standard-attributes.md:12]: Sure, and put a link to that item in this section.

Comment (louischan-oursky) [reply apis/admin-apis/update-users-standard-attributes.md:12]: @carmenlau in case you missed this issue.

**Comment by carmenlau** [comment]
Updated in #75

**Pull request closed without merging**